media-gfx/blender/blender-4.0.2-r1.ebuild

```diff
@@ -484,7 +522,7 @@ pkg_postrm() {
 
 	ewarn ""
 	ewarn "You may want to remove the following directory."
-	ewarn "~/.config/${PN}/${SLOT}/cache/"
+	ewarn "\~/.config/${PN}/${SLOT}/cache/"
```

Comment:
This shows up as a literal backslash in the post-merge message. Is there a specific reason to escape it?

Reply:
It silences the [shellcheck warning](https://www.shellcheck.net/wiki/SC2088) and makes it clear that it's no supposed to be expanded.